src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -158,6 +164,16 @@ public string GetNewSpeciesName()
         return speciesNameEdit.Text;
     }
 
+    public Color GetMembraneColor()
+    {
+        return membraneColorPicker.Color;
+    }
+
+    public float GetRigidity()
+    {
+        return (float)fluiditySlider.Value;
```

Comment:
This is *very* suspicious to be, why is this cast to float? Is the slider value a double? If it is something else than a double I doubt that this works correctly as the rigidity should be between 0 and 2 (I think).